mediation-json-to-xml/policy.bal

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import choreo/mediation;
+import ballerina/xmldata;
+import ballerina/mime;
+import ballerina/http;
+
+@mediation:InFlow
+public function jsonToXmlIn(mediation:Context ctx, http:Request req) returns http:Response|false|error|() {
+    json jsonPayload = check req.getJsonPayload();
+    xml? xmlPayload = check xmldata:fromJson(jsonPayload);
+
+    if xmlPayload is () {
+        http:Response errResp = new;
+        errResp.statusCode = http:STATUS_INTERNAL_SERVER_ERROR;
+        return errResp;
```

Comment:
You may use StatusCodeResponse record rather using the http:Response
https://ballerina.io/learn/by-example/http-resource-returns/

Reply:
Can't use this in this particular case since the return type signature doesn't accept `StatusCodeResponse`